frontend/src/components/common/button/button.tsx

```diff
@@ -1,12 +1,17 @@
 import { ButtonType } from 'common/enums/enums';
+import styles from './styles.module.scss';
 
 type Props = {
   label: string;
   type?: ButtonType;
+  useStyle?: string;
 };
 
-const Button: React.FC<Props> = ({ type = ButtonType.BUTTON, label }) => (
-  <button type={type}>{label}</button>
+const Button: React.FC<Props> = ({ type = ButtonType.BUTTON, label, useStyle = '' }) => (
+  <button
+    type={type}
+    className={`${styles.button} ${useStyle ? styles[useStyle] : ''}`}
```

Comment:
```suggestion
    className={getAllowedClasses(styles.button, styles[useStyle]}
```
use getAllowedClasses function (we already have it)